sdk/cosmos/azure-cosmos-benchmark/ctl/linkedin/run_benchmark.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+############################################################################
+## The parameters supported
+##  - ctl_endpoint (required)
+##  - ctl_key (required)
+##  - ctl_database (optional: default testdb)
+##  - ctl_collection (optional: default testCol)
+##  - ctl_throughput (optional: default 100,000)
+##  - ctl_concurrency (optional: default 50)
+##  - ctl_consistency_level (optional: default Session)
+##  - ctl_number_of_precreated_documents (optional: default 100,000)
+##  - ctl_number_of_operations (optional: default 1,000,000)
+##  - ctl_max_running_time_duration (optional: default 10 minutes)
+##  - ctl_printing_interval (optional: default 30 seconds)
+
+service_endpoint=$ctl_endpoint
+master_key=$ctl_key
+operation=LinkedInCtlWorkload
+number_Of_collection=1
+connection_mode=Direct
+protocol=Tcp
+
+if [ -z "$ctl_database" ]
+then
+database_name="testdb"
+else
+database_name=$ctl_database
+fi
+
+if [ -z "$ctl_collection" ]
+then
+collection_name="testCol"
+else
+collection_name=$ctl_collection
+fi
+
+if [ -z "$ctl_throughput" ]
+then
+throughput=100000
+else
+throughput=$ctl_throughput
+fi
+
+if [ -z "$ctl_concurrency" ]
+then
+concurrency=50
+else
+concurrency=$ctl_concurrency
+fi
+
+if [ -z "$ctl_consistency_level" ]
+then
+consistency_level=Session
+else
+consistency_level=$ctl_consistency_level
+fi
+
+if [ -z "$ctl_number_of_precreated_documents" ]
+then
+number_of_precreated_documents=100000
```

Comment:
Can we make default to 1M , that's what we wanted for LI workload right  ?

Reply:
For our workload we want it ~10M+. From our email thread with Shelly, we should be able to allocate a collection with 1M RUs, and load unto 5TB of data. We will have to experiment with loading a larger and larger set of documents, and see how it impacts the data load times

Changed the default value to 1M here.